development/cpp/configuring_an_ide.rst

```diff
@@ -270,29 +270,29 @@ Visual Studio Code
 
 - Now open cloned godot folder in VS Code (File > Open Folder...)
 
-In order to build the project, we need configurations files: *launch.json* and *tasks.json*.
+In order to build the project, we need two configuration files: *launch.json* and *tasks.json*.
 To create them:
 
-- Open Debug view (Ctrl + Shift + D) and select cogwheel with orange dot:
+- Open Debug view (Ctrl + Shift + D) and select cogwheel with an orange dot:
 
 .. image:: img/vscode_1_create_launch.json.png
 
-- Select *C++ (GDB/LLDB)* (it might be slightly differently called on macOS or Windows)
+- Select *C++ (GDB/LLDB)* (it might be named differently on macOS or Windows)
 
 - Update *launch.json* to match:
 
 .. image:: img/vscode_2_launch.json.png
 
-(Note that *godot.x11.tools.64* in "program" value might be differently called on macOS or Windows)
+(Note that *godot.x11.tools.64* in "program" value might be named differently on macOS or Windows)
 
-- Create *tasks.json* by starting Debug process (F5). VS Code will show dialog with *Configure Task* button. Tap it and select *Create tasks.json file from template*, then select *Others*
+- Create *tasks.json* by starting the Debug process (F5). VSCode will show a dialog with a *Configure Task* button. Tap it and select *Create tasks.json file from template*, then select *Others*
```

Comment:
"VS Code" is generally preferred to "VSCode" as a short form for "Visual Studio Code":

```suggestion
- Create *tasks.json* by starting the Debug process (F5). VS Code will show a dialog with a *Configure Task* button. Tap it and select *Create tasks.json file from template*, then select *Others*
```